targets/targets.json

```diff
@@ -1801,58 +1801,33 @@
         "supported_form_factors": ["ARDUINO"],
         "release_versions": ["5"],
         "config": {
-            "usb_tx": {
+            "stdio_uart_tx": {
                 "help": "Value: D8(default) or D1",
                 "value": "D8"
             },
-            "usb_rx": {
+            "stdio_uart_rx": {
                 "help": "Value: D2(default) or D0",
                 "value": "D2"
-            },
-            "stdio_uart": {
-                "help": "Value: UART_1(default) or UART_3",
-                "value": "UART_1",
-                "macro_name": "STDIO_UART"
             }
         }
     },
     "MBED_CONNECT_ODIN": {
         "inherits": ["MODULE_UBLOX_ODIN_W2"],
         "release_versions": ["5"],
         "config": {
-            "usb_tx": {
+            "stdio_uart_tx": {
                 "help": "Value: PA_9(default) or PD_8",
                 "value": "PA_9"
             },
-            "usb_rx": {
+            "stdio_uart_rx": {
                 "help": "Value: PA_10(default) or PD_9",
                 "value": "PA_10"
-            },
-            "stdio_uart": {
-                "help": "Value: UART_1(default) or UART_3",
-                "value": "UART_1",
-                "macro_name": "STDIO_UART"
             }
         }
     },
     "MTB_UBLOX_ODIN_W2": {
         "inherits": ["MODULE_UBLOX_ODIN_W2"],
-        "release_versions": ["5"],
-        "config": {
-            "usb_tx": {
-                "help": "Value PA_11",
-                "value": "PA_11"
-            },
-            "usb_rx": {
-                "help": "Value PA_13",
-                "value": "PA_13"
-            },
-            "stdio_uart": {
-                "help": "Value: UART_1",
-                "value": "UART_1",
-                "macro_name": "STDIO_UART"
-             }
-        }
+        "release_versions": ["5"]
```

Comment:
bad rebase ? these pins should be redefined here?

Reply:
No, as there is no "user choice" for this target, the default configuration in the PinName.h file should be sufficient ?
No need to overload the json file.

Reply:
I was wondering how this is different than from Odin target (line 1818 changes it to a new name) but here you remove it here, thus I asked.

Not clear the line 1831 removal of stdio_uart for odin also?

Reply:
Yes, STDIO_UART is not used any more in serial_api.c file